test/fixtures/requests/index.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
   },
   prepareSettings: {
     domain: require('./prepare-settings'),
+    noSignerEntries: require('./prepare-settings-no-signer-entires'),
```

Comment:
entires -> entries

Reply:
oops. Will fix the typo

Reply:
Thanks for taking the time to review the code.